config/constants.ts

```diff
@@ -52,6 +52,12 @@ export const oncoanalyserBucket: Record<AppStage, string> = {
   [AppStage.PROD]: 'org.umccr.data.oncoanalyser',
 };
 
+export const icav2PipelineCacheBucket: Record<AppStage, string> = {
+  [AppStage.BETA]: 'pipeline-dev-cache-503977275616-ap-southeast-2',
+  [AppStage.GAMMA]: 'pipeline-stg-cache-503977275616-ap-southeast-2',
+  [AppStage.PROD]: 'pipeline-prod-cache-503977275616-ap-southeast-2',
+};
+
```

Comment:
Nice! So these are all buckets that forward events to the respective accounts which then get ingested by filemanager?

Reply:
Yup. And, there will be more...